NSS-Spontaneous-Spoonbills/Sprint2
# Em ok computer form

Closes NOTHING

## Description of Proposed Changes

* Adds update computer form & URL
* Adds new computer form & URL
* Makes some minor additions to .gitignore
* Makes some clarification changes on the pull request template

## Steps to Test

```sh
git fetch --all
git checkout em_ok_computer_form
```

1. Start your virtual environment and run your server
1. Delete everything from migrations except __pycache__ and __init__.py
1. Delete your sqlite db
1. Make migrations and migrate
1. Navigate to this URL: ```http://127.0.0.1:8000/WorkforceManagement/computers/```
1. Verify the following:
  1. You can see the link for creating a new computer
  1. You are able to make a new computer using the form & it will show up in the computer list
    * Note: You should not see a "decomission date" field in the new computer form.
  1. You can see an update link beside each computer
  1. Clicking the update link will take you to a form that allows you to update the computer's information
    * Note: You should not see a field for changing the computer's purchase date, only it's decom date.
  1. After updating the computer, you can see the updated details when clicking on the link for that computer from the computer list.

## Impacted Areas in Application

**New Files**
WorkforceManagement/forms/Computer_Form.py
WorkforceManagement/forms/__init__.py

**Deleted Files**
WorkforceManagement/templates/WorkforceManagement/Computer_New.html

**Modified Files**
.gitignore
README.md
WorkforceManagement/models/Computer_Model.py
WorkforceManagement/templates/WorkforceManagement/Computer_List.html
WorkforceManagement/templates/WorkforceManagement/Computer_Update.html
WorkforceManagement/urls.py
WorkforceManagement/views/Computer_View.py
WorkforceManagement/views/__init__.py
docs/PULL_REQUEST_TEMPLATE.md

## Mentions @username

@obscure-shadow @JacobAndrewSmith92 @binauralbeat @CashewRose @kimberly-bird 

## Testing
